symbols/arm9.yml

```diff
@@ -901,6 +901,19 @@ arm9:
         r0: the effects bitvector to test
         r1: exclusive item effect ID
         return: bool
+    - name: ApplyGummiBoostsGround
```

Comment:
Does "Ground" refer to ground mode? If so, can we call this "ApplyGummiBoostsGroundMode"? "Ground" makes me think of the type.